TrackletAlgorithm/InputRouter.h

```diff
@@ -316,6 +337,4 @@ void InputRouter( const BXType bx
 }
 
 
-#endif
-
-
+#endif
```

Comment:
Could you please add a newline to the end of the file?

Reply:
applied to latest commit.

Reply:
should be fixed in 1055d5ee68c7c31bf35a9d2d940b5a9668b82483